src/coreclr/binder/inc/bindertypes.hpp

```diff
@@ -68,6 +68,7 @@ typedef enum __PEKIND
     peAMD64     = 0x00000004,
     peARM       = 0x00000005,
     peARM64     = 0x00000006,
+    peLOONGARCH64 = 0x00000007,
```

Comment:
Should I delete this for just only for windows?

Reply:
It's only used on windows,  guessed by `src/coreclr/binder/CMakeLists.txt` .

Reply:
Hi, @jkotas 
I merged all the latest PRs on my LoongArch64-linux machine and compiled it.
But now there are some errors within the lastes main branch caused by these files `src/coreclr/binder/assemblybindercommon.cpp`, `src/coreclr/binder/inc/bindertypes.hpp`, `src/coreclr/debug/createdump/datatarget.cpp`, `src/native/corehost/hostmisc/utils.cpp` and `src/coreclr/vm/ceeload.h`.

Now these files are not only used for windows.
Should I modify these files based on the latest main branch ?

Reply:
What are the errors that you are getting?

Reply:
The patch fixed all errors liking :

```
diff --git a/src/coreclr/binder/assemblybindercommon.cpp b/src/coreclr/binder/assemblybindercommon.cpp
index 86f3a2efb9f..830c673ea06 100644
--- a/src/coreclr/binder/assemblybindercommon.cpp
+++ b/src/coreclr/binder/assemblybindercommon.cpp
@@ -1447,8 +1447,6 @@ BOOL AssemblyBinderCommon::IsValidArchitecture(PEKIND kArchitecture)
         peARM;
 #elif defined(TARGET_ARM64)
         peARM64;
-#elif defined(TARGET_LOONGARCH64)
-        peLOONGARCH64;
 #else
         PORTABILITY_ASSERT("processArchitecture");
 #endif
diff --git a/src/coreclr/binder/inc/bindertypes.hpp b/src/coreclr/binder/inc/bindertypes.hpp
index 15a07b3d7b5..101ef3c8d95 100644
--- a/src/coreclr/binder/inc/bindertypes.hpp
+++ b/src/coreclr/binder/inc/bindertypes.hpp
@@ -68,7 +68,6 @@ typedef enum __PEKIND
     peAMD64     = 0x00000004,
     peARM       = 0x00000005,
     peARM64     = 0x00000006,
-    peLOONGARCH64 = 0x00000007,
     peInvalid   = 0xffffffff,
 } PEKIND;
 
diff --git a/src/coreclr/debug/createdump/datatarget.cpp b/src/coreclr/debug/createdump/datatarget.cpp
index d119498c2b2..7d2ab1973ff 100644
--- a/src/coreclr/debug/createdump/datatarget.cpp
+++ b/src/coreclr/debug/createdump/datatarget.cpp
@@ -65,8 +65,6 @@ DumpDataTarget::GetMachineType(
     *machine = IMAGE_FILE_MACHINE_ARM64;
 #elif HOST_X86
     *machine = IMAGE_FILE_MACHINE_I386;
-#elif HOST_LOONGARCH64
-    *machine = IMAGE_FILE_MACHINE_LOONGARCH64;
 #else
 #error Unsupported architecture
 #endif
@@ -77,7 +75,7 @@ HRESULT STDMETHODCALLTYPE
 DumpDataTarget::GetPointerSize(
     /* [out] */ ULONG32 *size)
 {
-#if defined(HOST_AMD64) || defined(HOST_ARM64) || defined(HOST_LOONGARCH64)
+#if defined(HOST_AMD64) || defined(HOST_ARM64)
     *size = 8;
 #elif defined(HOST_ARM) || defined(HOST_X86)
     *size = 4;
diff --git a/src/coreclr/vm/ceeload.h b/src/coreclr/vm/ceeload.h
index 5454787b8f5..e7dd5629a2c 100644
--- a/src/coreclr/vm/ceeload.h
+++ b/src/coreclr/vm/ceeload.h
@@ -87,8 +87,6 @@ class JITInlineTrackingMap;
 #define NATIVE_SYMBOL_READER_DLL W("Microsoft.DiaSymReader.Native.arm.dll")
 #elif defined(HOST_ARM64)
 #define NATIVE_SYMBOL_READER_DLL W("Microsoft.DiaSymReader.Native.arm64.dll")
-#elif defined(HOST_LOONGARCH64)
-#define NATIVE_SYMBOL_READER_DLL W("Microsoft.DiaSymReader.Native.loongarch64.dll")
 #endif
 
 typedef DPTR(JITInlineTrackingMap) PTR_JITInlineTrackingMap;

```

Reply:
OK, Thanks for guidance.